gslib/gcs_json_credentials.py

```diff
@@ -133,6 +133,12 @@ def GetCredentialStoreKey(credentials, api_version):
   return '-'.join(key_parts)
 
 
+def GetImpersonateServiceAccount():
```

Comment:
This does not cover the service account scenario.

We should be instead using https://github.com/GoogleCloudPlatform/gsutil/blob/3b61bf0e6188f65f78c72c79ea3cb69e9c61da4b/gslib/gcs_json_api.py#L314

But that method is inside gcs_json_api, which means we will have to add an API method that exposes this method.

Reply:
Fixed!